Dockerfile

```diff
@@ -1,75 +1,61 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 
 ENV DEBIAN_FRONTEND noninteractive
 ENV PYTHONUNBUFFERED 1
 
-RUN apt-get update -q
-
-# For python 3.6
-RUN apt-get install -yqq software-properties-common \
-  && add-apt-repository ppa:deadsnakes/ppa \
-  && apt-get update -q
-
-RUN apt-get update \
-  && apt-get install -y postgresql-client \
-    libproj-dev \
-    gdal-bin \
-    memcached \
-    libmemcached-dev \
-    build-essential \
-    python3.6 \
-    python3-pip \
-    python-virtualenv \
-    python3.6-dev \
-    git \
-    libssl-dev \
-    libpq-dev \
-    gfortran \
-    libatlas-base-dev \
-    libjpeg-dev \
-    libxml2-dev \
-    libxslt-dev \
-    zlib1g-dev \
-    python-software-properties \
-    ghostscript \
-    python3-celery \
-    python3-sphinx \
-    openjdk-9-jre-headless \
-    locales \
-    pkg-config \
-    gcc \
-    libtool \
-    automake
+RUN apt-get update -q &&\
```

Comment:
Codacy found an issue: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6150428)